code/game/objects/effects/effect_system/effects_foam.dm

```diff
@@ -66,9 +65,6 @@
 	L.adjust_fire_stacks(-2)
 	L.ExtinguishMob()
 
-/obj/effect/particle_effect/foam/firefighting/temperature_expose(datum/gas_mixture/air, exposed_temperature, exposed_volume)
-	return
```

Comment:
This is to stop fire from deleting it instantly.

Reply:
i removed fire melting foam to begin with